schema/deploy/tables/reporting_requirement.sql

```diff
@@ -48,8 +48,10 @@ comment on column cif.reporting_requirement.submitted_date is 'The date the repo
 comment on column cif.reporting_requirement.status is 'The status of the reporting requirement: on_track, late, completed, in_review';
 comment on column cif.reporting_requirement.comments is 'Comments about the reporting requirement';
 comment on column cif.reporting_requirement.certified_by is 'The CIF contact who certified the reporting requirement';
+comment on column cif.reporting_requirement.certified_by_professional_designation is 'The professional designation of the CIF contact who certified the reporting requirement';
```

Comment:
I don't think the contact is from the CIF team who certifies the reporting requirement here.
It's the industrial reporter who needs to fill out the name and professional designation of the person who assessed the job was done (professional engineer for equipment installed, CPA for expense reports, etc.)